tests/data/serialization/ndjson/test_metric.py

```diff
@@ -16,14 +16,29 @@ def test_metric():
     list(LBV1Converter.serialize(label_list))
 
 
-def test_custom_metric():
+def test_custom_scalar_metric():
     with open('tests/data/assets/ndjson/custom_scalar_import.json',
               'r') as file:
         data = json.load(file)
 
     label_list = NDJsonConverter.deserialize(data).as_list()
     reserialized = list(NDJsonConverter.serialize(label_list))
-    assert reserialized == data
+    assert json.dumps(reserialized,
+                      sort_keys=True) == json.dumps(data, sort_keys=True)
+
+    # Just make sure that this doesn't break
+    list(LBV1Converter.serialize(label_list))
+
+
+def test_custom_confusion_matrix_metric():
+    with open('tests/data/assets/ndjson/custom_confusion_matrix_import.json',
+              'r') as file:
+        data = json.load(file)
+
+    label_list = NDJsonConverter.deserialize(data).as_list()
+    reserialized = list(NDJsonConverter.serialize(label_list))
+    assert json.dumps(reserialized,
+                      sort_keys=True) == json.dumps(data, sort_keys=True)
```

Comment:
how does this test `test_custom_confusion_matrix_metric` ? it seems like it just deserialize and re-serializes a json file, then asserts that its equal to the file data?

Reply:
This tests the serialization and deserialization of the custom metric. Not the actual calculation.

Reply:
got it. I might recommend naming it something about the serialization, then but that just a nit!